Sources/SwiftProtobuf/Google_Protobuf_Any+Registry.swift

```diff
@@ -47,9 +46,12 @@ internal func typeName(fromURL s: String) -> String {
   return String(s[typeStart..<s.endIndex])
 }
 
+#if !os(WASI)
+import Dispatch
```

Comment:
Can you use `canImport` for these gates instead of the `os(WASI)` check?

I think I'd rather keep the import at the top gated also so it is less hidden.